packages/integration_test/ios/Classes/FLTIntegrationTestCase.m

```diff
@@ -0,0 +1,76 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// XCTest is weakly linked.
+#if __has_include(<XCTest/XCTest.h>)
```

Comment:
`FLTIntegrationTestCase` only works from a XCTest target (not from the app itself).